onnx/test/shape_inference_test.py

```diff
@@ -173,6 +194,34 @@ def _compare_value_infos(
                 "Unrecognized value info type in _compare_value_infos: ", str(vi_type)
             )
 
+    def _test_an_op(self, op_name, test_function_name) -> None:
+        """Given an operator name and test_function_name, run tests for op versions > 5.
+        :param op_name: The operator name.
+        :param test_function_name: The test function name for the op. It's used to form test functions.
+            The caller can use self.get_caller_function_name() to get the function name.
+        :return: None
+        """
+        versions = list(self.op_versions[op_name])
+        assert versions is not None
+        if len(versions) == 0:
+            # The version starts from 0 to now.
+            function_name = f"_internal_{test_function_name}"
+            test_func = getattr(self, function_name, None)
+            if test_func is None:
+                raise NotImplementedError(f"Cannot find test function {function_name}")
+            test_func(op_name=op_name)
+            return
+        for version in versions:
+            # Many versions found for this op.
+            if int(version) <= 5:
+                # Not sure how to fix the Reshape error in self._make_graph.
```

Comment:
Create an issue to track this if it needs to be fixed, then link it here?

Reply:
Done: https://github.com/onnx/onnx/issues/5289